packages/graphql-transformers-e2e-tests/src/__tests__/ConnectionsWithAuthTests.e2e.test.ts

```diff
@@ -232,46 +239,45 @@ beforeAll(async () => {
         console.log(`Using graphql url: ${GRAPHQL_ENDPOINT}`);
 
         // Get the details of the created user pool
-        const userPoolIdSelector = outputValueSelector(ResourceConstants.OUTPUTS.AuthCognitoUserPoolIdOutput)
-        const userPoolClientIdSelector = outputValueSelector(ResourceConstants.OUTPUTS.AuthCognitoUserPoolJSClientOutput)
-        const userPoolId = userPoolIdSelector(finishedStack.Outputs);
-        const userPoolClientId = userPoolClientIdSelector(finishedStack.Outputs);
-
+        // const userPoolIdSelector = outputValueSelector(ResourceConstants.OUTPUTS.AuthCognitoUserPoolIdOutput)
```

Comment:
Can we get rid of this instead of commenting?

Reply:
yup